### PR TITLE
added test for the mongoose model validation

### PR DESCRIPTION
**Pull request by neighbor983**
simple mongoose validation. Used mocha can relook at using jasmine later if that is what we want to do.

### CODE REVIEW

**Comment by dghalbr** [comment]
Hmmm. Change the naming scheme to something like user-spec.js and I need to chew on this when I get time later.

File: test/userTest.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+var User = require('../server/models/user.js');
+
+describe('User', function() {
+
+    describe('User Mongoose Model', function() {
+
+        it('should be invalid if email is empty', function(done) {
+            var user = new User();
+
+            user.validate(function(err) {
```
Comment (dghalbr) [inline test/userTest.js]: I need an explanation for how this is working since there is no validate function in user.js lol.

**Comment by neighbor983** [comment]
The required field is validation. if the newUser doesn't have an email it
will not work.

**Comment by dghalbr** [comment]
I understand that, but there is no validate method on the user object as far as I know.

**Pull request closed without merging**